src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +154,20 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function lastNameMale()
+    {
+        return parent::lastName();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function lastNameFemale()
+    {
+        return parent::lastName() . 'а';
+    }
 }
```

Comment:
could you override the `lastName` formatter just like the `middleName` one, so that it chooses randomly a male or female name?

Reply:
HI! After rebase i've found out that there is already appropriate ru_RU lastName generator. Thus this particular PR can be closed )